gensim/corpora/wikicorpus.py

```diff
@@ -164,18 +168,20 @@ def remove_file(s):
     return s
 
 
-def tokenize(content):
+def tokenize(content, token_min_len=TOKEN_MIN_LEN, token_max_len=TOKEN_MAX_LEN, lower=True):
     """
     Tokenize a piece of text from wikipedia. The input string `content` is assumed
     to be mark-up free (see `filter_wiki()`).
 
+    set token_min_len, token_max_len as length thresholds for individual tokens
```

Comment:
Start with capital letter + full stop at the end + variables in backticks for varbatim formatting, ``token_min_len``.
Here and elsewhere.